src/styles/devibeans.css

```diff
@@ -1,6 +1,8 @@
-/*
-    devibeans (dark)
-    Created by Farzad Sadeghi (http://github.com/terminaldweller)
+/*!
+    Theme: devibeans(dark)
+    Author: @terminaldweller
```

Comment:
You can spell your name out here (if you'd like)... the author field is really where we want the Github IDs incase we need to ping you in the future regarding your theme.  This is ok though if it's what you prefer.